presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -64,14 +66,16 @@ private static QueryRunner createQueryRunner()
                 Optional.empty());
 
         queryRunner.execute(noPushdownFilter(queryRunner.getDefaultSession()),
-                "CREATE TABLE lineitem_ex (linenumber, orderkey, partkey, suppkey, ship_by_air, is_returned, ship_day, ship_month, ship_timestamp, order_timestamp, keys, nested_keys, flags) AS " +
+                "CREATE TABLE lineitem_ex (linenumber, orderkey, partkey, suppkey, ship_by_air, is_returned, ship_day, ship_month, ship_timestamp, order_timestamp, discount_real, tax_real, keys, nested_keys, flags) AS " +
                         "SELECT linenumber, l.orderkey, partkey, suppkey, " +
                         "   IF (linenumber % 5 = 0, null, shipmode = 'AIR') AS ship_by_air, " +
                         "   IF (linenumber % 7 = 0, null, returnflag = 'R') AS is_returned, " +
                         "   IF (linenumber % 4 = 0, null, CAST(day(shipdate) AS TINYINT)) AS ship_day, " +
                         "   IF (linenumber % 6 = 0, null, CAST(month(shipdate) AS TINYINT)) AS ship_month, " +
                         "   IF (linenumber % 3 = 0, null, CAST(shipdate AS TIMESTAMP)) AS ship_timestamp, " +
                         "   IF (l.orderkey % 3 = 0, null, CAST(o.orderdate AS TIMESTAMP)) AS order_timestamp, " +
+                        "   IF (linenumber % 3 = 0, null, CAST(discount AS REAL)) AS discount_real, " +
```

Comment:
- make sure to use different moduli for different columns to avoid have all nulls rows; here 3 is used for ship_timestamp, order_timestamp and discount_real; 5 is used for ship_by_air and tax_real.
- consider using only prime numbers and apply these to orderkey instead of linenumber. That's because the range of values for orderkey is [1, 60000] and for linenumber it is [1, 7] (See the output of `SHOW STATS FOR lineitem`).